sources/packages/web/src/store/modules/institution/actions.ts

```diff
@@ -57,4 +58,16 @@ export const actions: ActionTree<InstitutionLocationState, RootState> = {
       await InstitutionService.shared.getMyInstitutionLocationsDetails();
     context.commit("setMyInstitutionLocationsDetailsState", resultComment);
   },
+
+  /**
+   * Set the institution user details
+   * during the institution setup.
+   * @param context action context.
+   */
+  setInstitutionUserDetailsOnSetup(
+    context: ActionContext<InstitutionLocationState, RootState>,
+  ): void {
+    const userState = { isInstitutionSetupUser: true } as UserStateForStore;
```

Comment:
I was thinking shouldn't `isInstitutionSetupUser` is set to not true, when the intended logic is achieved, since it is stored in the store,  they will be able to access the `INSTITUTION_CREATE` URL  again after they have the institution right?

Reply:
Thanks for bringing this point @ann-aot . As discussed we assume that once the institution setup is complete on click of submit on the same page, the store variables are now created for institution which will eventually overwrite the institution user object in the store.

Again as discussed, tested the same and found the assumption is right.

![image](https://user-images.githubusercontent.com/54600590/233222601-4503a7a6-c999-4591-a0d6-693cedb629af.png)

![image](https://user-images.githubusercontent.com/54600590/233222636-e3594421-c077-4c47-a80e-c63f74f44bc2.png)